include/roboteam_world/ros_handler.h

```diff
@@ -1,47 +1,35 @@
 #pragma once
 
-#include "ros/ros.h"
-
-#include "std_srvs/Empty.h"
-
-#include "roboteam_msgs/DetectionFrame.h"
-#include "roboteam_msgs/World.h"
-#include "roboteam_msgs/Tracker.h"
-
+#include "messages_robocup_ssl_detection.pb.h"
+#include "World.pb.h"
 #include "roboteam_world/world/world_dummy.h"
 #include "roboteam_world/world/filtered_world.h"
-
+#include "Subscriber.h"
+#include <Publisher.h>
```

Comment:
We might want to be more strict on including with <> vs "" for visibility? Typically you only use "" for files defined within a project.